contracts/RMRK/extension/tokenProperties/RMRKTokenProperties.sol

```diff
@@ -0,0 +1,144 @@
+// SPDX-License-Identifier: Apache-2.0
+
+pragma solidity ^0.8.16;
+
+import "./IRMRKTokenProperties.sol";
+
+/**
+ * @title RMRKTokenProperties
+ * @author RMRK team
+ * @notice Smart contract of the RMRK Token properties module.
+ */
+contract RMRKTokenProperties is IRMRKTokenProperties {
```

Comment:
I'm a bit confused by this class, I think it should be a mock since it has no gating.
The strategy we use for extensions is to have:
1. An interface
2. A core implementation, non opinionated. Here all the variables are defined as private with public getters and only internal setters. This is the one a custom implementer would extend.
3. A mock, with gated or not external setters which call the internal ones, this is the one we use for tests.
4. In some cases, an implementation, which adds gates. (I wouldn't add it for this one)

I feel this class is mixing 2 and 3. I'm open to discussion.

Reply:
Ok so in this case you would prefer to call this RMRKTokenPropertiesMock ?

Reply:
Yeap that would be the easiest.